gatling-http/src/main/scala/io/gatling/http/request/builder/HttpParam.scala

```diff
@@ -21,5 +21,5 @@ import io.gatling.core.session.Expression
 sealed trait HttpParam
 final case class SimpleParam(key: Expression[String], value: Expression[Any]) extends HttpParam
 final case class MultivaluedParam(key: Expression[String], values: Expression[Seq[Any]]) extends HttpParam
-final case class ParamSeq(seq: Expression[Seq[(String, Any)]]) extends HttpParam
+final case class ParamSeq(seq: Expression[Seq[(Any, Any)]]) extends HttpParam
```

Comment:
Why? While values can be considered as not Strings (eg String representations of numbers), keys are always Strings.